### ISSUE TITLE
automate adding new language

### ISSUE THREAD

**Issue by vvalchev** [preslavmihaylov/default-setups#2]
First, I loved the idea of automating everything and I believe you can automate everything, so I'm starting to add some ideas:

To read my current keyboard layout settings I did:
```
$ defaults read com.apple.HIToolbox AppleEnabledInputSources                                                                     (
        {
        InputSourceKind = "Keyboard Layout";
        "KeyboardLayout ID" = 252;
        "KeyboardLayout Name" = ABC;
    },
        {
        InputSourceKind = "Keyboard Layout";
        "KeyboardLayout ID" = 19529;
        "KeyboardLayout Name" = "Bulgarian - Phonetic";
    },
   ...
```

So to add a new keyboard input I have to type from the console:
```
defaults write com.apple.HIToolbox AppleEnabledInputSources -array-add '<dict><key>InputSourceKind</key><string>Keyboard Layout</string><key>KeyboardLayout ID</key><integer>19529</integer><key>KeyboardLayout Name</key><string>Bulgarian - Phonetic</string></dict>'
```

**Comment by presmihaylov** [comment]
This is great! 

Thanks a lot for the help. I wasn't aware of the defaults command. :)

**Comment by presmihaylov** [comment]
Implemented via b858e93
